scipy/stats/_morestats.py

```diff
@@ -856,15 +856,21 @@ def ppcc_plot(x, a, b, dist='tukeylambda', plot=None, N=80):
 
 def _log_mean(logx):
     # compute log of mean of x from log(x)
-    return special.logsumexp(logx, axis=0) - np.log(len(logx))
+    res = special.logsumexp(logx, axis=0) - math.log(logx.shape[0])
+    return res
```

Comment:
```suggestion
    return special.logsumexp(logx, axis=0) - math.log(logx.shape[0])
```
In the past, I didn't used to like using `math` operations when there were NumPy equivalents (even if math was faster), but considering the Array API standard and a desire to preserve dtypes, it really comes in handy when working with Python scalars.